pygmt/src/text.py

```diff
@@ -157,15 +156,17 @@ def text_(
     # Ensure inputs are either textfiles, x/y/text, or position/text
     if position is None:
         kind = data_kind(textfiles, x, y, text)
```

Comment:
It it worth protecting against passing both textfiles and x/y/text to data_kind()? Just affects what error the user gets.
```suggestion
        if (x is not None or y is not None) and textfiles is not None:
            raise GMTInvalidInput( "Provide either position only, or x/y pairs, or textfiles.")
        kind = data_kind(textfiles, x, y, text)
```